power-control-x86/src/power_control.cpp

```diff
@@ -1628,6 +1631,7 @@ int main(int argc, char* argv[])
 {
     const std::string P0ThermTrip = "P0_THERMTRIP_L";
     const std::string P1ThermTrip = "P1_THERMTRIP_L";
+    const std::string MonRSMRST   = "MON_RSMRST_L";
```

Comment:
is MON_RSMRST_L set in the device tree?

Reply:
https://gerrit-git.amd.com/c/ESE/ec/BMC/openbmc/+/976348
add MON_RSMRST_L to Volcano DTS files